docs/rules/no-confusing-arrow.md

```diff
@@ -32,6 +30,8 @@ Examples of **correct** code for this rule:
 /*eslint no-confusing-arrow: "error"*/
 /*eslint-env es6*/
 
+var x = (a) => 1 ? 2 : 3;
```

Comment:
`var x = (a) => 1 ? 2 : 3;` is still an incorrect case.

Reply:
Do you squash merge? I mean, do I need to amend my commit then force push, or just add another commit?

Reply:
We use squash merge, so both amending the last commit and adding new commit are OK.